runtime/src/accounts_db.rs

```diff
@@ -4638,17 +4630,16 @@ impl AccountsDb {
 
         let mut measure_shrink_all_candidates = Measure::start("shrink_all_candidate_slots-ms");
         let num_candidates = shrink_slots.len();
-        let shrink_candidates_count: usize = self.thread_pool_clean.install(|| {
+        let shrink_candidates_count = shrink_slots.len();
+        self.thread_pool_clean.install(|| {
             shrink_slots
                 .into_par_iter()
-                .map(|(slot, slot_shrink_candidates)| {
+                .for_each(|(slot, slot_shrink_candidates)| {
                     let mut measure = Measure::start("shrink_candidate_slots-ms");
-                    self.do_shrink_slot_stores(slot, slot_shrink_candidates.values());
+                    self.do_shrink_slot_stores(slot, std::iter::once(&slot_shrink_candidates));
```

Comment:
maybe it makes sense to change the signature of self.do_shrink_slot_stores to do_shrink_slot_store(slot, store) instead of the plurals to clarify

Reply:
I'll get there. It goes on and on up and down the call stack. `ShrinkCandidates` is the focus of this pr.